test/metabase/query_processor/middleware/constraints_test.clj

```diff
@@ -16,7 +17,7 @@
   (testing "if it is *truthy* add the constraints"
     (is (= {:middleware  {:add-default-userland-constraints? true},
             :constraints {:max-results           @#'qp.constraints/max-results
-                          :max-results-bare-rows @#'qp.constraints/default-max-results-bare-rows}}
+                          :max-results-bare-rows @#'qp.constraints/max-results-bare-rows}}
            (add-default-userland-constraints
```

Comment:
```suggestion
                          :max-results-bare-rows (qp.constraints/max-results-bare-rows)}}
           (add-default-userland-constraints
```

`@#'qp.constraints/max-results-bare-rows` just gives you the value of `qp.constraints/max-results-bare-rows`, it's a way to get around Clojure complaining about it being `^:private` (which is totally fine in tests). It's not private anymore, so you no longer need to do this.

`:max-results-bare-rows` shouldn't be equal to the value of `qp.constraints/max-results-bare-rows` tho, because `qp.constraints/max-results-bare-rows` is a function as created by `defsetting`. `:max-results-bare-rows` should be equal to the value of calling that function

Reply:
It think you also need to change the line above for `max-results`.